js/src/views/Dapps/dappsStore.js

```diff
@@ -44,13 +45,27 @@ export default class DappsStore {
   _registryAppsIds = null;
 
   constructor (api) {
+    super();
+
     this._api = api;
 
     this.readDisplayApps();
     this.loadExternalOverlay();
     this.subscribeToChanges();
   }
 
+  static get (api) {
+    if (!instance) {
+      instance = new DappsStore(api);
+    }
+
+    if (api && !instance._api) {
```

Comment:
Not sure why we are doing this - when we create it we should properly pass in the api. (There is one in the app, no reason why it cannot be passed in)